src/pages/extending-agent-app/index.mdx

```diff
@@ -6,14 +6,18 @@ tagline: "Discover how you can extend the LiveChat Agent App."
 desc: "You can extend the LiveChat Agent App with widgets, buttons, and other UI elements. Build the native-feel LiveChat integrations."
 ---
 
+# Introduction
+
+This document presents how you can change and extend the LiveChat App and its interface, so it is personalized and follow your needs. You can do it using [Chat Actions](/extending-agent-app#chat-actions) or [App locations](/extending-agent-app#app-locations).
```

Comment:
```suggestion
This document presents how you can change and extend the LiveChat App and its interface, so it is personalized and follow your needs. You can do it by mounting/putting/placing your app in one of the available [App locations](/extending-agent-app#app-locations).
```

Reply:
mounting/putting/placing - not sure which is best or maybe there's a better option

Reply:
by configuring your app to be displayed in one of the available app locations" - longish but most true

Reply:
seems long, I agree...but...I like it as it is because, like you wrote above, most true. I will change it this way :)